charts/tidb-cluster/templates/tidb-initializer-job.yaml

```diff
@@ -28,21 +28,28 @@ spec:
         - -c
         - |
 {{ tuple "scripts/_initialize_tidb_users.py.tpl" . | include "helm-toolkit.utils.template" | indent 10 }}
+        {{- if or .Values.tidb.passwordSecretName .Values.tidb.initSql }}
         volumeMounts:
+        {{- if .Values.tidb.passwordSecretName }}
```

Comment:
For readability, I think nested condition statement should have different indent level with the outer one.